spec/yelp/error_spec.rb

```diff
@@ -40,5 +40,19 @@
       end        
     end
 
+    context 'when the API returns the error description' do
+      let(:response_body) { '{"error": {"text": "One or more parameters are invalid in request", "id": "INVALID_PARAMETER", "field": "limit", "description": "Limit maximum is 20"}}' }
+
+      it 'should expose more details about the invalid parameter' do
+        begin
```

Comment:
I was trying to follow the standard stablished in the other tests. Does it make sense to have this one be the only different?